roles/kubernetes-apps/cluster_roles/tasks/main.yml

```diff
@@ -75,6 +75,33 @@
     - node_webhook_crb_manifest.changed
   tags: node-webhook
 
+- name: Write vsphere-cloud-provider ClusterRole manifest
+  template:
+    src: "vsphere-rbac.yml.j2"
+    dest: "{{ kube_config_dir }}/vsphere-rbac.yml"
+  register: vsphere_rbac_manifest
+  when:
+    - rbac_enabled
+    - cloud_provider is defined
+    - cloud_provider == 'vsphere'
+    - kube_version | version_compare('v1.9.0', '>=')
+  tags: vsphere
+
+- name: Apply vsphere-cloud-provider ClusterRole
```

Comment:
I think we'll want to do a delete/apply here versus apply. 

It's pretty unlikely that the role will change, but if it does an apply will not work.

Reply:
Agreed. I will just check if ClusterRole doesn't exists before creation.